### PR TITLE
Tweaks for Jetpack 8.4

### PR DESCRIPTION
**Pull request by dchymko**
## Description
A couple of small UI changes to hide some of the Instant Search elements for now

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).

## Steps to Test
1. Pull down PR locally or to sandbox
1. Navigate to Jetpack setting screen (`/wp-admin/admin.php?page=jetpack#/performance`)
1. Under the Search section under the "Add Jetpack Search Widget", confirm that this card/button no longer appears: "Help visitors quickly find answers with highly relevant instant search results and powerful filtering | Upgrade "

### CODE REVIEW

**Comment by nickdaugherty** [comment]
Didn't get to this today, will look first thing tomorrow.

**Comment by nickdaugherty** [comment]
@dchymko this hides the promotions, but I think Instant Search is still visible as a product, right? From internal discussions, it was my understanding that we wanted to hide all references to it for the time being.

Also, we should base the disabling on a constant so that we can enable it on a per-site basis.

**Comment by dchymko** [comment]
> @dchymko this hides the promotions, but I think Instant Search is still visible as a product, right? From internal discussions, it was my understanding that we wanted to hide all references to it for the time being.
> 
> Also, we should base the disabling on a constant so that we can enable it on a per-site basis.

I don't think we have an easy way of hiding the actual toggle until this is done:
https://github.com/Automattic/jetpack/issues/15405

I'll add the constant check to enable these for select sites

**Comment by dchymko** [comment]
Added the constant check in https://github.com/Automattic/vip-go-mu-plugins/pull/1548/commits/28c3153b3d12e87a5a1a7b03bd5927154e0251c0

File: vip-jetpack/vip-jetpack.php [hunk 1]
```diff
@@ -348,3 +348,14 @@ function wpcom_vip_disable_jetpack_email_no_recaptcha( $is_enabled ) {
 
 	return $modules;
 } );
+
+/**
+ * Hide promotions/upgrade cards for now except for sites that we want to opt into Instant Search
+ */
+add_filter( 'jetpack_show_promotions', function ( $is_enabled ) {
+	if ( defined( 'VIP_GO_JETPACK_ENABLE_INSTANT_SEARCH' ) && VIP_GO_JETPACK_ENABLE_INSTANT_SEARCH ) {
```
Comment (nickdaugherty) [inline vip-jetpack/vip-jetpack.php:356]:
```suggestion
	if ( defined( 'VIP_JETPACK_ENABLE_INSTANT_SEARCH' ) && true === VIP_JETPACK_ENABLE_INSTANT_SEARCH ) {
```

**Comment by nickdaugherty** [comment]
r888-stacks

**Pull request merged**